python/mxnet/executor.py

```diff
@@ -311,212 +309,3 @@ def debug_str(self):
             self.handle, ctypes.byref(debug_str)))
         return py_str(debug_str.value)
 
-def _split_input_slice(batch_size, work_load_list):
```

Comment:
It's probably better to keep them there since these protected functions are really utility functions and users are using them.
Alternatively, add `import * from executor_manager` in executor.py.